pybleau/app/io/tests/test_plotting_roundtrip.py

```diff
@@ -90,14 +93,9 @@ def test_round_trip_df_plotter_cmap_scatter_type(self):
                               z_col_name="Col_4",
                               plot_config=config, plot_title="Plot 6")
```

Comment:
There's a duplicated plot name here (same `Plot 6` name on line 84). I noticed you changed some of the others, so this one may need the same attention.

Reply:
Done.